package/MDAnalysis/core/groups.py

```diff
@@ -2725,19 +2725,22 @@ def asunique(self, sorted=False):
         Examples
         --------
 
+           >>> import MDAnalysis as mda
+           >>> from MDAnalysis.tests.datafiles import PSF, DCD
```

Comment:
Since you are importing MDAnalysis as mda, you should use `mda` rather than `MDAnalysis` going forward.

Reply:
Thanks for the prompt review and feedback @orionarcher! I just tried using `mda` in my terminal and this was the result:
```
$ python
Python 3.11.0 | packaged by conda-forge | (main, Oct 25 2022, 06:12:32) [MSC v.1929 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import MDAnalysis as mda
C:\Users\Xu Hong Chen\Projects\mdanalysis\package\MDAnalysis\topology\TPRParser.py:161: DeprecationWarning: 'xdrlib' is depreca
ted and slated for removal in Python 3.13
  import xdrlib
>>> from mda.tests.datafiles import PSF, DCD
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'mda'
```
How might you go about doing this? It does work when I use `MDAnalysis` instead.

Reply:
Ahh, nvm, I may be wrong here.

Reply:
All good; appreciate you taking the time to address everything!